requirements.txt

```diff
@@ -13,3 +13,7 @@ us
 geopandas
 pytest
 black
+xlrd
+isort
+importmagic
+epc
```

Comment:
How would you feel about pinning the versions we're using? We may have slightly different python environment philosophies (which is totally fine).  Also, definitely doesn't have to happen (or maybe we could provide for both ways) but we don't use conda at all in our other two repos.  Do you think it's worth keeping repo setup the same across all repos?

Reply:
Totally should -- this requirements.txt is a bit "wip" as I add more things that I use in my python env. 

We'll update this in another issue!

Reply:
Another comment -- I should separate my personal preferneces/setup/dependencies for working on this project from what is absolutely necessary to run the scrapers.

Will do when I resolve the issue in #3